cms/templates/import.html

```diff
@@ -127,6 +191,19 @@ <h3 class="title">${_("Success")}</h3>
       </form>
     </article>
 
+  %if library:
+    <aside class="content-supplementary" role="complimentary">
+      <div class="bit">
+          <h3 class="title-3">${_("Why import a library?")}</h3>
+          <p>${_("You might want to update an existing library to a new version, or replace an existing library entirely. You might also have developed a library outside of Studio.")}</p>
+      </div>
+
+      <div class="bit">
+          <h3 class="title-3">${_("Note: Library content is not automatically updated in courses")}</h3>
```

Comment:
@Kelketek You saved me, thanks! There's nothing worse than staring at wording I suggested and really disliking it :-)